awscli/customizations/globalargs.py

```diff
@@ -90,6 +95,21 @@ def resolve_cli_connect_timeout(parsed_args, session, **kwargs):
     arg_name = 'connect_timeout'
     _resolve_timeout(session, parsed_args, arg_name)
 
+def detect_migration_breakage(parsed_args, remaining_args, argument_table, session, **kwargs):
+    if parsed_args.migrate_v2:
+        url_params = [param for param in remaining_args if param.startswith('http://') or param.startswith('https://')]
+        uni_print("AWS CLI v2 MIGRATION WARNING: By default, AWS CLI v2 returns all output through your operating system’s default pager program. See https://docs.aws.amazon.com/cli/latest/userguide/cliv2-migration-changes.html#cliv2-migration-output-pager.\n")
```

Comment:
I'm still on the fence for whether this one is useful, since users cannot address it while still on V1, right?